cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -239,6 +240,22 @@ class LeadImpl extends ServerImpl with Lead
     }
   }
 
+  private def checkAuthProvider(props: Properties): Unit = {
+    var authP = props.getProperty(Attribute.AUTH_PROVIDER)
+    if (authP == null) {
+      authP = props.getProperty(Attribute.SERVER_AUTH_PROVIDER)
+    }
```

Comment:
Ok, but rather than relying on above information, better to handle here that may just require an extra else/if block.